FirebaseFirestore.podspec

```diff
@@ -79,4 +77,20 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
         Firestore/core/src/firebase/firestore/util/config.h.in > \
         Firestore/core/src/firebase/firestore/util/config.h
   CMD
+
+  s.subspec 'abseil-cpp' do |ss|
+    ss.source_files = [
+      'Firestore/third_party/abseil-cpp/**/*.{h,cc}'
+    ]
+    ss.exclude_files = [
+      'Firestore/third_party/abseil-cpp/**/*_test.{h,cc}',
+    ]
+
+    # NB: don't include absl/**/*.h, as that would include absl/*/internal/*.h.
+    ss.public_header_files = 'Firestore/third_party/abseil-cpp/absl/*/*.h'
```

Comment:
This doesn't seem right, and it causes the CocoaPods generated umbrella header to include the abseil headers. That in turn causes the swift build failure that @paulb777 pointed out.

You can see this by opening `FirebaseFirestore-umbrella.h` after doing a pod update on this branch.